apps/web/src/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationActions.tsx

```diff
@@ -72,6 +115,30 @@ const SeeAdvertisementAction = ({
     </Link>
   );
 };
+export type SupportActionProps = ActionProps;
+
+const SupportAction = ({ show }: SupportActionProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  if (!show) {
+    return null;
+  }
+
+  return (
+    <Link
+      href={paths.support()}
+      data-h2-font-weight="base(700)"
+      data-h2-color="base(tm-blue.dark)"
+    >
+      {intl.formatMessage({
+        defaultMessage: "Get support",
+        id: "rXdaZW",
+        description: "Link text to direct a user to the support page",
+      })}
+    </Link>
+  );
+};
 
 export interface DeleteActionProps extends ActionProps {
```

Comment:
opening delete dialog throws this
might have been an existing issue?

![image](https://user-images.githubusercontent.com/40485260/219791289-622aa821-a4a9-44f5-8375-a826cd525327.png)

Reply:
Definitely a preexisting issue. Fixed in [fix issue with nested p tags in application dialogs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5744/commits/42f9289e919319fd6e0623f98b52f3802edfafec)